pom.xml

```diff
@@ -60,12 +60,12 @@
 		<maven.compiler.target>11</maven.compiler.target>
 		<maven.compiler.source>11</maven.compiler.source>
 
-		<wicket.version>9.16.0</wicket.version>
+		<wicket.version>10.0.0-M2</wicket.version>
 		<jackson.version>2.16.1</jackson.version>
 		<slf4j.version>2.0.12</slf4j.version>
 		<junit.version>5.10.2</junit.version>
-		<servlet-api.version>4.0.4</servlet-api.version>
-		<jetty.version>9.4.54.v20240208</jetty.version>
+		<servlet-api.version>6.0.0</servlet-api.version>
```

Comment:
Wicket uses Servlet API 5.x. Do we require 6.x for Wiquery ?

Reply:
ah no, i will go to 5 instead